rust/gui-client/src-tauri/src/client/logging.rs

```diff
@@ -77,7 +77,12 @@ pub(crate) fn setup(directives: &str) -> Result<Handles> {
 
 #[tauri::command]
 pub(crate) async fn clear_logs() -> StdResult<(), String> {
-    clear_logs_inner().await.map_err(|e| e.to_string())
```

Comment:
Since this returns to Tauri, we hadn't previously gotten any log in the log files or terminal, just in the JS dev console